### PR TITLE
Feat discovergy calls

### PR DESCRIPTION
**Pull request by missgreenwood**
Implements [APP-88,](https://buzzn-team.atlassian.net/browse/APP-88), [APP-84](https://buzzn-team.atlassian.net/browse/APP-84), [APP-106](https://buzzn-team.atlassian.net/browse/APP-106) and [APP-95](https://buzzn-team.atlassian.net/browse/APP-95).

### CODE REVIEW

**Review by clemensmanert — CHANGES_REQUESTED** [review]
measuring unit missing

File: routes/consumption_history.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                                   datetime.min.time()).timestamp() * 1e3)
+    begin = request.args.get('begin', default=start, type=int)
+    end = request.args.get('end', default=None, type=int)
+    tics = request.args.get('tics', default='one_hour', type=str)
+    return begin, end, tics
+
+
+@IndividualConsumptionHistory.route('/individual-consumption-history',
+                                    methods=['GET'])
+def individual_consumption_history():
```
Comment (clemensmanert) [inline routes/consumption_history.py:41]: measuring unit missing in name or docu

File: routes/consumption_history.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+
+def read_parameters():
+    """ Use the given parameters. """
+
+    start = round(datetime.combine(datetime.now(),
+                                   datetime.min.time()).timestamp() * 1e3)
+    begin = request.args.get('begin', default=start, type=int)
+    end = request.args.get('end', default=None, type=int)
+    tics = request.args.get('tics', default='one_hour', type=str)
```
Comment (clemensmanert) [inline routes/consumption_history.py:35]: why not an enum?

File: routes/consumption_history.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+logger = logging.getLogger(__name__)
+IndividualConsumptionHistory = Blueprint('IndividualConsumptionHistory',
+                                         __name__)
+GroupConsumptionHistory = Blueprint('GroupConsumptionHistory', __name__)
+
+
+def read_parameters():
+    """ Use the given parameters. """
+
+    start = round(datetime.combine(datetime.now(),
```
Comment (clemensmanert) [inline routes/consumption_history.py:31]: What does this do? And Why?

File: routes/disaggregation.py [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+
+logger = logging.getLogger(__name__)
+IndividualDisaggregation = Blueprint('IndividualDisaggregation', __name__)
+GroupDisaggregation = Blueprint('GroupDisaggregation', __name__)
+
+
+def read_parameters():
+    """ Use the given parameters. """
+    start = round((datetime.now() - timedelta(hours=48)).timestamp() * 1e3)
```
Comment (clemensmanert) [inline routes/disaggregation.py:21]: What is 1e3? And Why?

**Review by clemensmanert — COMMENTED** [review]
We could use http codes from  flask_status status.

File: .pylintrc [hunk 5]
```diff
@@ -151,6 +151,8 @@ disable=print-statement,
         wrong-import-position,
         too-few-public-methods,
         ungrouped-imports, 
+        fixme, 
+        too-many-function-args,
```
Comment (clemensmanert) [inline .pylintrc]: Consider using 
`#pylint: disable=unused-variable`
on the function where you need more.

File: routes/consumption_history.py (hunk 2 above)
Comment (missgreenwood) [reply routes/consumption_history.py:35]: To return a list, you mean? AFAIK, in Python you rather return tuples if possible (their members are numbered just the same).

File: routes/disaggregation.py (hunk 4 above)
Comment (missgreenwood) [reply routes/disaggregation.py:21]: What you get by timestamp() is the seconds precision, but the Discovergy API requires milliseconds. * 1e3 means ^(-3), so you get the ms precision and round it afterwards to have an integer value as requested by their API. I clarified this with a comment.

**Pull request merged**